chart/templates/_helpers.yaml

```diff
@@ -638,7 +638,7 @@ Create the name of the cleanup service account to use
   - -c
   - |
     CONNECTION_CHECK_MAX_COUNT=0 AIRFLOW__LOGGING__LOGGING_LEVEL=ERROR exec /entrypoint \
-    airflow jobs check --job-type SchedulerJob --hostname $(hostname)
+    airflow jobs check --job-type SchedulerJob --local
```

Comment:
This needs to be conditional on 2.4.0 or later.

Reply:
Yep, I did it in a previous commit and removed it. thanks. Re added it in dc45752cb037fb68e06de3e8e49e7ad59caefa00